### ISSUE TITLE
Respect xunit analyzers

### ISSUE THREAD

**Issue by Meir017** [kblok/puppeteer-sharp#161]
we have close to 200 asserts that trigger `warning xUnit2000: The literal or constant value xxx should be the first argument in the call to 'Assert.Equal(expected, actual)'`

**Comment by kblok** [comment]
Yeah. My next task is getting rid of all the warnings we have in units. We missed that on reviews...
